src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -19,7 +19,8 @@
 
     <!-- When building from source, ensure the RID we're building for is part of the RID graph -->
     <AdditionalRuntimeIdentifiers Condition="'$(DotNetBuildFromSource)' == 'true'">$(AdditionalRuntimeIdentifiers);$(OutputRID)</AdditionalRuntimeIdentifiers>
-    <ServicingVersion>5</ServicingVersion>
+    <ServicingVersion>9</ServicingVersion>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
I don't understand this change. Why is it being bumped to 9?

@wfurt @ericstj @ViktorHofer is this an expected change for a RID addition?

Reply:
I admit this was a bit of a shot in the dark. I only noticed this snippet seemed to be added when RIDs had been added previously (and subsequently removed in ulterior commits). I've assumed it forces `Microsoft.NETCore.Platforms` to be built, with a distinct servicing version.

Reply:
See this other RID PR for Fedora, which I am about to merge BTW:

https://github.com/dotnet/runtime/pull/74383

I suspect that if we merge this Rocky RID on the same servicing release as the  Fedora one, the number should be bumped only once, and to the next number.

But please correct me if I'm wrong, @wfurt / @ViktorHofer / @ericstj .

Reply:
Agreed. I only wonder whether it should be aligned with the .NET service release version number.

Reply:
For these packages we increment them each time we ship them.  We don't skip numbers.

We could make this be shipped always, in which case it could just take the product version, but that's not the way it's set up to work at the moment.

Reply:
Ok so since I just merged https://github.com/dotnet/runtime/pull/74383, and based on @ericstj answer, I think this PR can remove the package authoring changes in `Microsoft.NETCore.Platforms.csproj`.

Can you please revert that file's changes, @mhlindstr?

Reply:
Yup, will do. Thanks!